### PR TITLE
Adjust cmake configuration to install things properly

### PR DESCRIPTION
**Pull request by morphis**
To have a real use we need to install the library, public headers and a proper pkg-config file.

### CODE REVIEW

**Comment by kanavin** [comment]
lgtm

File: libwds/CMakeLists.txt [hunk 1]
```diff
@@ -2,7 +2,26 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -fvisibility=hidden")
 set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} -std=c99 -Wall")
```
Comment (pozdnyakov) [inline libwds/CMakeLists.txt]: why -std=c99 is needed?

Comment (morphis) [inline libwds/CMakeLists.txt]: Not sure but looks like that is there already for some time.

Comment (kanavin) [inline libwds/CMakeLists.txt]: It was there from the beginning. gcc defaults to 'GNU dialect of ISO C90', so setting to C99 is a better choice.

**Comment by morphis** [comment]
Just found one minor issue with this.

**Comment by morphis** [comment]
Fixed that now. The pkg-config file is now properly setup. Had that still locally pending but not included here.

File: data/wds.pc.in [hunk 2]
```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${exec_prefix}/include
+ 
+Name: @CMAKE_PROJECT_NAME@
+Description: Library to build Miracast/WiDi-enabled applications
```
Comment (alexshalamov) [inline data/wds.pc.in]: Description: Library to build WiFi Display applications

**Comment by alexshalamov** [comment]
lgtm

**Pull request merged**